### ISSUE TITLE
Migrate to `@electron/remote`

### ISSUE THREAD

**Issue by sindresorhus** [wulkano/Kap#1089]
It blocks us from upgrading Electron and we're left on an old Electron version...

https://github.com/electron/remote/blob/main/README.md#migrating-from-remote

**Comment by karaggeorge** [comment]
Yeah that was the biggest blocker from migrating the cropper to the new style like the rest of the windows. We use remote a lot, so it was taking a while switching it. I had an ongoing branch, I can try to find it and push it as a draft as a starting point
